BackofficeBundle/Resources/views/BackOffice/Underscore/fullPagePanelView._tpl.twig

```diff
@@ -0,0 +1,15 @@
+{% include 'OpenOrchestraBackofficeBundle:BackOffice:Underscore/Jarvis/header._tpl.twig' %}
+
+<div role="container">
+    <div id="panelView">
+        <ul class="nav nav-tabs" role="tablist" id="superboxTab">
+
```

Comment:
These empty line is useless. Have you tried the xhtml reduced expression <ul class="..." /> ?